FrontEnd/components/Tweet/ListFeild.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <div v-if="loading" class="h-screen flex justify-center items-center">
+    <div v-if="props.loading" class="h-screen flex justify-center items-center">
```

Comment:
we can access variables directly defined for props when using in template, in this case it will be 
 
 v-if="loading"

Reply:
Updated props.loading return loading as suggested. Please review the latest commit